libmozevent/utils.py

```diff
@@ -146,6 +146,15 @@ class AsyncRedis(object):
     async def connect():
         if AsyncRedis.redis.get(None) is None:
             AsyncRedis.redis.set(
-                await aioredis.from_url(os.environ["REDIS_URL"], decode_responses=False)
+                await aioredis.from_url(
+                    os.getenv("REDIS_TLS_URL", None),
```

Comment:
Nit: you already checked it's not None, so you can just do `os.environ["REDIS_TLS_URL"]`